usagov/vote-gov-drupal
# VOTE-1490 Add config and string translations for Russian translation

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-1490

## Description (optional)
Add configuration and string translations for Russian translation

## Deployment and testing (required, if applicable)
### Post-deploy steps
1. `lando retune`

### Testing steps
1. Visit http://vote-gov.lndo.site/admin/config/regional/language
2. Confirm Russian language is created and disabled
